### ISSUE TITLE
Transaction does not balance when one amount omited

### ISSUE THREAD

**Issue by hoshsadiq** [niknetniko/intellij-beancount#16]
Seems when leaving out one of the amounts in a simple transaction causes an error of "Transaction does not balance" when it is valid. For example, below transaction causes error underline for `"self" "Deposit/Withdrawal"` though it is [perfectly valid](https://beancount.github.io/docs/beancount_language_syntax.html#transactions).

```
2021-08-04 * "self" "Deposit/Withdrawal"
  Assets:Investment   100.00 GBP
  Assets:Current
```

**Comment by niknetniko** [comment]
Yeah, the short term solution will probably be to allow disabling this check, as it is currently very basic: it sums up all numbers in the transaction and requires that sum to be zero. It doesn't account for currency, nor prices or any of the other "advanced" stuff, and does also not account for interpolation, as you've noticed.

This built-in check is mostly an experiment, since I found calling bean-check from the plugin to be a bit slow, but I think that's the way to go, since it would be a lot of work to reimplement whole parts of beancount in the plugin.

**Comment by hoshsadiq** [comment]
Aah okay, that makes sense.

I wonder if running bean-check through Jython would be faster than invoking it directly. Only supports Python 2.7 so not sure if it would be work but if beancount is backwards compatible it might be worth trying.
